wandb/sdk/internal/job_builder.py

```diff
@@ -105,9 +109,22 @@ def disable(self) -> bool:
     def disable(self, val: bool) -> None:
         self._disable = val
 
-    def _set_logged_code_artifact(
+    def _handle_server_artifact(
         self, res: Optional[Dict], artifact: "ArtifactRecord"
     ) -> None:
+        try:
```

Comment:
nit: this reads weird to me, I'd do: if type==job; try/except, elif type==code...

Reply:
done